packages/tslint-config-loris/index.js

```diff
@@ -0,0 +1,115 @@
+module.exports = {
+    // Add additional rules from https://github.com/buzinas/tslint-eslint-rules
+    'rulesDirectory': 'node_modules/tslint-eslint-rules/dist/rules',
+    'rules': {
+        // TypeScript specific
+        'no-empty-interface': true,
+        'no-internal-module': true,
+        'typedef-whitespace': [
+            true,
+            {
+                'call-signature': 'nospace',
+                'index-signature': 'nospace',
+                'parameter': 'nospace',
+                'property-declaration': 'nospace',
+                'variable-declaration': 'nospace'
+            }
+        ],
+        'array-type': [true, 'array'],
+        'class-name': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#possible-errors
+        'trailing-comma': [true, {multiline: 'never', singleline: 'never'}],
+        'no-conditional-assignment': true,
+        'no-console': [
+            true,
+            // Need manually specify all disallowed functions.
+            'assert',
+            'dir',
+            'error',
+            'info',
+            'log',
+            'time',
+            'timeEnd',
+            'trace',
+            'warn'
+        ],
+        'no-constant-condition': [true, {checkLoops: false}],
+        'no-control-regex': true,
+        'no-debugger': true,
+        'no-duplicate-case': true,
+        'no-empty-character-class': true,
+        'no-ex-assign': true,
+        'no-extra-semi': true,
+        'no-extra-boolean-cast': true,
+        'no-inner-declarations': [true, 'both'],
+        'no-invalid-regexp': true,
+        'no-irregular-whitespace': true,
+        'no-regex-spaces': true,
+        'no-sparse-arrays': true,
+        'no-unexpected-multiline': true,
+        'no-unsafe-finally': true,
+        'use-isnan': true,
+        'valid-typeof': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#best-practices
+        'curly': true,
+        'triple-equals': true,
+        'no-arg': true,
+        // Disable this rule, because it isn't configurable
+        // https://github.com/palantir/tslint/issues/1088
+        // https://github.com/palantir/tslint/issues/2107
+        //'no-empty': true,
+        'no-eval': true,
+        'no-multi-spaces': true,
+        'no-duplicate-variable': [true, "check-parameters"],
+        'no-string-throw': true,
+        'no-unused-expression': false,
+        'radix': true,
+        'no-unused-variable': true,
+        'no-use-before-declare': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#variables
+        'array-bracket-spacing': [true, 'never'],
+        'brace-style': [true, '1tbs'],
+        'variable-name': [true, 'ban-keywords'],
+        'eofline': true,
+        // TODO(ikokostya): Find easy way to disable rules in subdirectory.
+        'ter-indent': false, // [true, 4, {SwitchCase: 1}],
```

Comment:
I disabled this rule, because it got false positive results. I try enable this rule again now and it works as expected for the following versions:

```
├── tslint@5.7.0 
└── tslint-eslint-rules@4.1.1
```

So, I suggest to enable it.